src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bag/RitualBagEntity.kt

```diff
@@ -64,7 +64,7 @@ class RitualBagEntity : ThrowableEntity, IRendersAsItem {
         val recipe = RitualBagItem.getRecipe(item)
         val inv = Inventory(5)
         recipe.ifPresent {
-            it.applyEffects(world, position, inv)
+            it.applyEffects(world, position, inv, ItemStack.EMPTY)
```

Comment:
Maybe it's better to create a `PlaySoundEffect`, so that we can change the visual and audible feedback per ritual? Also in regard of future `negative_effects`

See also upper comment.

Reply:
See #49 for follow up